packages/html2react/processors/script.tsx

```diff
@@ -0,0 +1,20 @@
+import { Processor, Element } from "../types";
+import Script from '@frontity/components/script';
+
+const script: Processor = {
+  test: node => node.type === "element" && node.component === "script",
```

Comment:
We need to add a name (I guess `"script"` it's fine) and a priority, which I think it should be at least 20 so other people can overwrite this.